x-pack/plugins/infra/public/alerting/log_threshold/components/expression_editor/criterion_preview_chart.tsx

```diff
@@ -184,15 +189,17 @@ const CriterionPreviewChart: React.FC<ChartProps> = ({
     if (!isGrouped) {
       return series;
     }
-
+    if (filterSeriesByGroupName && filterSeriesByGroupName.length) {
```

Comment:
@fkanout Does it mean we get the data for all the groups from BE and filter it on FE? Wasn't it possible to only filter on the BE?